elastic/elasticsearch
# support arm for shibboleth-idp

Work in progress. 

diff from fork: https://github.com/jakelandis/shibboleth-idp-dockerized/compare/7f8b92b4319dc7bfb6bb011fe37198170efaa667...master

To test on arm device

1) update the platform (I will clean up how we build the image)
2) ./gradlew :x-pack:qa:saml-idp-tests:javaRestTest --info

Note - I think we have some outstanding clean up issues with these running containers, so it might fail if not starting from a clean slate. Just run again (or manually clean up the running docker instances). 

Also note ... building from a linux base image instead of the upstream image i forked since we will eventually want to update/upgrade and the upstream image appears to no longer be getting updates.
